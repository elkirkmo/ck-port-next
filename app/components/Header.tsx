type HeaderProps = {
  name: string;
  location: string;
};

const Header = ({ name, location }: HeaderProps) => {
  return (
    <header>
      <h1>{name}</h1>
      <h2>Development | Documentary | Digital Filmmaking</h2>
      <h3>Based in {location}</h3>
    </header>
  );
};

export default Header;
