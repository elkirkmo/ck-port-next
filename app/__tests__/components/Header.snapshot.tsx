/**
 * @jest-environment jsdom
 */
import { render } from '@testing-library/react';
import Header from '../../components/Header';

it('renders Header Component with name and description', () => {
  const { container } = render(
    <Header name="test testerson" description="test test test" />
  );
  expect(container).toMatchSnapshot();
});
it('renders Header Component with name and description and location', () => {
  const { container } = render(
    <Header
      name="test testerson"
      description="test test test"
      location="Kissimme FL"
    />
  );
  expect(container).toMatchSnapshot();
});
