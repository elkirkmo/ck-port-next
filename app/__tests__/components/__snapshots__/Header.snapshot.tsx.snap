// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header Component with name and description 1`] = `
<div>
  <header
    class="text-right pr-20 py-10 text-white bg-gray-500/35 w-lvw"
  >
    <h1
      class="text-8xl"
    >
      test testerson
    </h1>
    <h2
      class="text-2xl"
    >
      test test test
    </h2>
  </header>
</div>
`;

exports[`renders Header Component with name and description and location 1`] = `
<div>
  <header
    class="text-right pr-20 py-10 text-white bg-gray-500/35 w-lvw"
  >
    <h1
      class="text-8xl"
    >
      test testerson
    </h1>
    <h2
      class="text-2xl"
    >
      test test test
    </h2>
    <h3
      class="text-lg"
      data-testid="location-h3"
    >
      Based in 
      Kissimme FL
    </h3>
  </header>
</div>
`;
